src/main/java/org/reaktivity/nukleus/maven/plugin/internal/generate/VariantFlyweightGenerator.java

```diff
@@ -344,10 +378,55 @@ public MemberFieldValueConstantGenerator addMember(
         }
     }
 
+    private static final class MissingFieldPlaceholderConstantGenerator extends ClassSpecMixinGenerator
+    {
+        private int missingFieldValue;
+        private boolean hasMissingFieldValue;
+
+        private MissingFieldPlaceholderConstantGenerator(
+            ClassName thisType,
+            TypeSpec.Builder builder)
+        {
+            super(thisType, builder);
+        }
+
+        public MissingFieldPlaceholderConstantGenerator addMember(
+            AstType memberType,
+            int missingFieldValue)
+        {
+            if (AstType.LIST32.equals(memberType) || AstType.LIST8.equals(memberType))
+            {
+                if (missingFieldValue != 0)
+                {
+                    hasMissingFieldValue = true;
+                    this.missingFieldValue = missingFieldValue;
+                }
+            }
+            return this;
+        }
+
+        @Override
+        public TypeSpec.Builder build()
+        {
+            if (hasMissingFieldValue)
+            {
```

Comment:
Can this be replaced by `if (missingFieldValue != 0)` and remove `hasMissingFieldValue` ?